docs/overview/project.rst

```diff
@@ -0,0 +1,128 @@
+
+You can check the status of current connections `here <../status.html#capture>`_. For more details on how autofill works you can go to the tutorial for :doc:`creating an analysis <../tutorials/create>`.
+
+Additionally, there is a possibility for uploading files (e.g. configuration files) as well as providing a URL from which the files are copied and stored automatically.
+
+.. TODO link to file upload tutorial
+
+Versioning
+~~~~~~~~~~
+
+Upon creation of an analysis on CAP it will be assigned a unique identifier. Every time the analysis is edited the new version will be stored as an update to the previous version of the analysis through the identifier system. This will enable references to intermediate analysis steps in the analysis notes, allow keeping track of the analysis and monitoring as well as undoing changes.
+
+Authorisation/Access Control
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Authorization on CAP is managed by CERN Single Sign-On, therefore restricting login to CERN members and restricting collaboration access to the collaboration you a member of.
+
+Due to the sensitive nature of research data and analysis information - especially in early stages of an analysis - accessibility of analysis information is subject to permissions set by the creator of an analysis and the researchers involved.
+
+When starting a new analysis, the creator of the analysis draft record will be the only one able to view and edit the record. He can grant read or edit rights to researchers in the analysis team or working group.
+
+As soon as the analysis is submitted, the analysis will be shared with the collaboration meaning its members will acquire rights to view the analysis. Editing rights will remain as they were for the draft.
+A draft version can be submitted any time. There is no drawback in keeping a draft until the analysis is published other than that analysis content is hidden from members of your collaboration.
+
+.. note::
+
+	- only CERN members have access to CAP login
+	- only collaboration members have access to a collaboration's area, can create analyses and can see shared analyses
+	- only a certain collaboration's members have access to this collaborations analyses
+	- only members granted specific rights can see or edit a draft version of an analysis
+	- only the creator can see or edit an analysis with default permission settings
+
+2. Search and Retrieve
+----------------------
+
+The search capability of CAP can help users find both archived and on-going analyses they have access to that contain the information specified in the search.
+
+Search capability
+~~~~~~~~~~~~~~~~~
+
+Using the search bar at the top of the page or the dedicated search page that comes with it, users can search through their own and all shared analyses within their collaboration, past or on-going. Based on feedback by collaboration physicists filters will be created to enable search for specific content.
```

Comment:
maybe add sth like: please check it out and let us know what is needed to make the search more helpful...